Cakes-Co/index.html

```diff
@@ -1,16 +1,118 @@
 <!DOCTYPE html>
 <html lang="en">
-
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
     <title>Responsive Cake webpage</title>
     <!-- Add a link to your css file here -->
-</head>
+    <link rel="stylesheet" href="style.css" />
+  </head>
 
-<body>
-    <!-- Add your markup here -->
-</body>
+  <body>
+    <div class="layout">
+      <!-- HEADER SECTION -->
+      <header>
+        <div class="logo-hamburger-container">
+          <img
+            src="/Cakes-Co/images/cupcake-svg-download.jpg"
+            width="100px"
+            height="100px"
+            class="cupcake-logo"
+            alt="cake-picture"
+          />
+          <div class="hamburger-menu"></div>
+        </div>
+        <p>The best cakes in town delivered to your door</p>
+      </header>
+      
+
+      <!-- NAVIGATION SECTION -->
+      <nav>
+        <div class="flex-nav">
+          <a href="#" class="home">HOME</a>
+          <a href="#" class="cakes">CAKES</a>
+          <a href="#" class="order-in">ORDER IN</a>
+          <a href="#" class="lessons">LESSONS</a>
+          <a href="#" class="about">ABOUT</a>
+        </div>
+      </nav>
+     
+
+      <!-- MAIN SECTION -->
+      <main>
+        <div>
+          <h1>Welcome</h1>
+          <p class="welcome-text">
+            Welcome to [Your Cake Website]! Whether you're a baking pro or just
+            love cake, you're in the right place. We've got delicious recipes,
+            expert tips, and amazing cake designs. Find your perfect cake here
+            and let's make your sweet dreams a reality!
+          </p>
+        </div>
+        <img
+          src="https://st4.depositphotos.com/10614052/25239/i/450/depositphotos_252391082-stock-photo-sweet-chocolate-cake-on-wooden.jpg  "
```

Comment:
Linking to a website on another site like this is called image hotlinking. It is considered a bad practice, because you are using another website's bandwidth and because the owner of the second website could delete or even overwrite the image (perhaps with an...unfortunate image). 

Here's some more info https://www.hostinger.co.uk/tutorials/hotlinking